pyanaconda/modules/storage/devicetree/utils.py

```diff
@@ -35,23 +35,35 @@
 log = get_module_logger(__name__)
 
 
+def is_supported_filesystem(fmt_type):
+    """Is the filesystem supported?
+
+    :param fmt_type: a name of the formatting type
+    :return: True or False
+    """
+    fmt = get_format(fmt_type)
+
+    if not fmt.type:
+        return False
+
+    # Check if the filesystem is unsupported by Anaconda.
+    if fmt.type in ("ntfs", "tmpfs"):
```

Comment:
Btrfs was removed intentionally? It makes sense to support "btrfs as filesystem" not only as a volume manager, but it might be worth doing this in a separate commit for clarity.

Reply:
Explained in a new commit.